src/adaptors/merkl/index.js

```diff
@@ -11,6 +11,15 @@ const networks = {
   60808: 'bob',
 };
 
+// Protocols that should not be listed under Merkl
+// as they already have their own adapters.
+const protocolsBlacklist = [
```

Comment:
I've added this because a pool cannot be listed twice
I've checked for all of them that they weren't returned by the old endpoint